src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ingroovy/GroovyStepExecutionCompiler.java

```diff
@@ -0,0 +1,49 @@
+package org.jenkinsci.plugins.workflow.cps.steps.ingroovy;
+
+import groovy.lang.GroovyClassLoader;
+import groovy.lang.GroovyShell;
+import hudson.Extension;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowExecution;
+import org.jenkinsci.plugins.workflow.cps.CpsGroovyShellFactory;
+
+/**
+ * Used to compile CPS-transformed code that does not run in the sandbox,
+ * but unlike {@linkplain CpsFlowExecution#getTrustedShell() trusted shell}, code
+ * loaded here will not automatically become visible to untrusted code.
+ *
+ * @author Kohsuke Kawaguchi
+ */
+@Extension
+public class GroovyStepExecutionCompiler {
+//    private GroovyShell sh;
```

Comment:
:bug: I really hate stumbling across commented out code like this where I don't know if it's actually valid, why it's there, why it's commented out, etc...so I'd strongly appreciate if it was removed. =)